### PR TITLE
Eliminate API conversion popup when loading in Unity 2017.1

### PR DESCRIPTION
**Pull request by davidkline-ms**
Add UNITY_2017_1_OR_NEWER blocks (and modify some existing 2017_2 blocks) to eliminate the API conversion popup when loading the MRTK in Unity 2017.1

There isn't an issue for this, it is an extension of #1921.

### CODE REVIEW

File: Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs [hunk 1]
```diff
@@ -367,12 +367,18 @@ public void OnInputDown(InputEventData eventData)
...
+                SupportedInputInfo.PointerPosition))
+#else
+                SupportedInputInfo.Position))
+#endif // UNITY_2017_1_OR_NEWER
             {
                 // The input source must provide positional data for this script to be usable
                 return;
             }
-#endif
+#endif // UNITY_2017_2_OR_NEWER
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: is this right?

File: Assets/HoloToolkit/UX/Prefabs/Text.meta [hunk 2]
```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 59c43794cfe522246aaa1d4a04cadf68
```
Comment (StephenHodgson) [inline Assets/HoloToolkit/UX/Prefabs/Text.meta]: What's this from?

Comment (davidkline-ms) [reply Assets/HoloToolkit/UX/Prefabs/Text.meta]: I missed adding this file as part of #2052

File: Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs (hunk 1 above)
Comment (davidkline-ms) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: nope. i fixed it in https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2061/commits/fb3fc5086606be1a5e3c9aacc3d01e6cc843187e

Comment (keveleigh) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: This matches line 358: https://github.com/davidkline-ms/MixedRealityToolkit-Unity/blob/fb3fc5086606be1a5e3c9aacc3d01e6cc843187e/Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs#L358

Comment (StephenHodgson) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: image

Comment (keveleigh) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: image

**Comment by StephenHodgson** [comment]
Do we have a PR for dev branch?

File: Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs (hunk 1 above)
Comment (keveleigh) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: Also, your red lines don't match up. There's an `#endif` between them

Comment (StephenHodgson) [reply Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/HandDraggable.cs]: Looks good for me now.

**Pull request merged**